test/dotnet-publish.Tests/PublishDesktopTests.cs

```diff
@@ -123,6 +123,25 @@ public class PublishDesktopTests : TestBase
             }
         }
 
+
+        [WindowsOnlyFact]
+        public async Task DesktopApp_WithRuntimes_PublishedSplitPackageAssets()
+        {
+            var testInstance = TestAssetsManager.CreateTestInstance(Path.Combine("..", "DesktopTestProjects", "DesktopAppWithRuntimes"))
```

Comment:
One option is to create your own `TestAssetsManager` which points to `DesktopTestProjects`.